src/components/Layers/Layer.scss

```diff
@@ -11,6 +11,7 @@
     visibility: visible;
     position: absolute;
     width: 100%;
+    z-index: 9999;
```

Comment:
this seems dangerous, especially for dropdowns and callouts inside layers?

Reply:
I agree, but the z-index has to be here so I will change him to 10

Reply:
Ok :)